models/DataObject/ClassDefinition/Data/Block.php

```diff
@@ -166,14 +166,14 @@ public function getDataForResource($data, $object = null, $params = [])
      * @param null|Model\DataObject\AbstractObject $object
      * @param mixed $params
      *
-     * @return string
+     * @return array|null
```

Comment:
not true, should be mixed since it could be anything

Reply:
PR: #3710